package.json

```diff
@@ -31,7 +31,7 @@
     "dependencies": {
         "comb": "~1.0.0",
         "comb-proxy": "~1.0.0",
-        "commander": ">=0.5.1",
+        "commander": "~2.9.0",
```

Comment:
I'm not sure about consumers but this restriction seems fairly non-passive. Must this be set to this version?